homeassistant/components/growatt_server/sensor/__init__.py

```diff
@@ -136,33 +81,38 @@ async def async_setup_entry(
     async_add_entities(entities, True)
```

Comment:
```suggestion
    async_add_entities(entities)
```
After config_entry_first_refresh we already have a state